datapackage_pipelines_mojp/common/processors/sync.py

```diff
@@ -99,17 +102,24 @@ def _filter_row(self, row, resource_descriptor):
                     if row[lang_field]:
                         for lang, value in row[lang_field].items():
                             if lang in iso639.languages.part1:
-                                new_doc["{}_{}".format(lang_field, lang)] = value
+                                new_doc["{}_{}".format(
+                                    lang_field, lang)] = value    
+                                new_doc["title_lc"] = {"title_{}_lc".format(lang): title.lower() for lang, title in row["title"].items()}
                             else:
-                                raise Exception("language identifier not according to iso639 standard: {}".format(lang))
+                                raise Exception(
+                                    "language identifier not according to iso639 standard: {}".format(lang))
                     # delete the combined json lang field from the new_doc
                     del new_doc[lang_field]
+                # add lower case titles
```

Comment:
This will change in order to handle _Nonetype_ titles (fail not recognized by tests)